lib/suggestions.js

```diff
@@ -9,8 +9,23 @@ import { Range } from 'atom'
 
 function resolveModule(textEditor, module) {
     const basedir = path.dirname(textEditor.getPath())
+
+    const moduleDirectory = [ 'node_modules' ]
+    const webpackConfigs = atom.config.get('js-hyperclick.webpackConfigs')
+    webpackConfigs.forEach(configFileName => {
+        try {
+            const config = require(path.join(atom.project.getPaths()[0], configFileName))
```

Comment:
I was thinking `fs.existsSync`, but apparently it's deprecated now. You're right. `fs.accessSync` does throw, which is a really stupid design. I don't know what the point is of having a check like that when it throws.

The next best solution is to just constrain the try/catch to the `require()` call.

```js
let config
try {
    config = require(...)
} catch (e) {
}

if (config && ...
```